src.ts/account.ts

```diff
@@ -6,7 +6,7 @@ import { Action, transfer, createAccount, signTransaction, deployContract,
 import { FinalExecutionOutcome, FinalExecutionStatusBasic } from './providers/provider';
 import { Connection } from './connection';
 import {base_decode, base_encode} from './utils/serialize';
-import { PublicKey } from './utils/key_pair';
+import {BlsPublicKey, PublicKey} from './utils/key_pair';
```

Comment:
Do we have docs somewhere on what's the diff between BlsPublicKey and regular PublicKey?

Reply:
It's not really used in nearlib. Rather it just holds data without actually doing anything.